src/inspector/inspectorhandler.cpp

```diff
@@ -636,7 +638,38 @@ static QList<InspectorCommand> s_commands{
                      [](InspectorHandler*, const QList<QByteArray>&) {
                        MZGlean::initialize();
                        return QJsonObject();
-                     }}};
+                     }},
+
+    InspectorCommand{
+        "set_subscription_start_date",
+        "Changes the start date of the subscription", 1,
+        [](InspectorHandler*, const QList<QByteArray>& arguments) {
+          qint64 newCreatedAtTimestamp = arguments[1].toLongLong();
+
+          // get sub data json from settings
+          QByteArray subscriptionData =
+              SettingsHolder::instance()->subscriptionData();
+          QJsonDocument doc = QJsonDocument::fromJson(subscriptionData);
+          QJsonObject obj = doc.object();
+          QJsonObject subObj = obj["subscription"].toObject();
+          qlonglong createdAt =
+              subObj["created"].toVariant().toLongLong() * 1000;
```

Comment:
Why is this line needed, since createdAt is overwriten in line 659?

Reply:
it's not 🙂